packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -22,11 +22,29 @@ import { cssModules } from 'bpk-react-utils';
 
 import STYLES from './bpk-badge.scss';
 
+export const BADGE_TYPES = {
+  WARNING: 'warning',
+  SUCCESS: 'success',
+  DESTRUCTIVE: 'destructive',
+  LIGHT: 'light',
+  INVERSE: 'inverse',
+  OUTLINE: 'outline',
```

Comment:
Lately we've started matching the keys and values, because it makes these easier to consume in Flow. E.g. in buttons we have:

```
export const BUTTON_TYPES = {
  primary: 'primary',
  secondary: 'secondary',
  destructive: 'destructive',
  featured: 'featured',
};
```

Then when we Flow type it we can do this:

```
export type ButtonType: $Keys<typeof BUTTON_TYPES>,
```

It's a workaround to get around Flow's current inability to have `$Values`.

Another benefit is that we can use it in PropTypes:

```
 type: PropTypes.oneOf(Object.keys(BUTTON_TYPES))
```

(I know `Object.values` exists but the browser support isn't good enough yet).